ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/IntegrationFlowView.java

```diff
@@ -98,9 +98,11 @@ public void clickRandomTrash() {
         this.getElementRandom(Element.DELETE).shouldBe(visible).click();
     }
 
-    public void clickAddStepLink(int pos) {
+    public void clickAddStepLink(int pos) throws InterruptedException {
+        Thread.sleep(2000);
         if (isExpanded()) {
             $(Button.Collapse).click();
+            Thread.sleep(2000);
```

Comment:
use `TestUtils.sleepIgnoreInterrupt` and you don't have to mess with the exception